packages/ckeditor5-image/docs/_snippets/features/build-image-source.js

```diff
@@ -8,8 +8,10 @@
 import ClassicEditor from '@ckeditor/ckeditor5-build-classic/src/ckeditor';
 
 import ImageResize from '@ckeditor/ckeditor5-image/src/imageresize';
+import ImageInsert from '@ckeditor/ckeditor5-image/src/imageinsert';
```

Comment:
Pedantical note – you added `ImageInsert` import between the two already existing ones, but then you used it as the first one (when pushing to `builtinPlugins`). Could you fix the order so it's identical?